src/content/docs/infrastructure/manage-your-data/data-instrumentation/host-integrations-metrics.mdx

```diff
@@ -13,9 +13,9 @@ redirects:
 This feature is currently in beta.
 </Callout>
 
-## New Relic Integrations Metrics [#integrations-metrics-table]
+Our infrastructure agent and our on-host integrations have traditionally reported as sample data: [metrics as metadata attached to events](/docs/data-apis/understand-data/new-relic-data-types/#metric-event-attributes). We have made these metrics also available as [dimensional metrics](/docs/data-apis/understand-data/new-relic-data-types/#dimensional-metrics), which is a better format for metric-type data in allowing for more in-depth analysis. 
```

Comment:
Mmmm this sentence is a little weird. How about: "Historically, our infrastructure agent and our on-host integrations have reported [metrics as sample metadata attached to events](/docs/data-apis/understand-data/new-relic-data-types/#metric-event-attributes)."